.gitlab-ci.yml

```diff
@@ -46,6 +46,8 @@ test:smoketests:
 test:acceptancetests:
   stage: test
   image: ubuntu:20.04
+  tags:
+    - hetzner-amd-beefy
```

Comment:
The tests are failing with `E0508 08:21:14.208027    8393 memcache.go:265] couldn't get current server API group list: Get "https://docker:41195/api?timeout=32s": EOF - error from a previous attempt: read tcp 172.20.0.7:60516->172.20.0.6:41195: read: connection reset by peer`